### PR TITLE
[2.5]Backport fleet airgap fix

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

**Comment by cbron** [comment]
https://github.com/rancher/rancher/issues/29327

File: chart/templates/service.yaml [hunk 1]
```diff
@@ -13,5 +13,9 @@ spec:
     targetPort: 80
     protocol: TCP
     name: http
+  - port: 443
+    targetPort: 444
+    protocol: TCP
+    name: https-internal
```
Comment (cbron) [inline chart/templates/service.yaml:19]: This is different from other PR, was there an additional PR that added this stuff into master earlier ?

Comment (StrongMonkey) [reply chart/templates/service.yaml:19]: @cbron Yeah, there are two commits made

**Comment by StrongMonkey** [comment]
@cbron I added https://github.com/rancher/rancher/commit/0390ff5bc542936c93969624c3644ddefb139b52 in this PR too. This backport commit modified the same code path so I would like to put them together

**Comment by StrongMonkey** [comment]
Also contains fix for https://github.com/rancher/rancher/issues/29652

**Review by cbron — DISMISSED** [review]
Ok, so it's backporting the following PRs in order: 
* https://github.com/rancher/rancher/pull/29502/files
* https://github.com/rancher/rancher/pull/29655/files
* https://github.com/rancher/rancher/pull/29663/files

It would be nice next time for them to all be done with separate PRs. One way we track if a given PR has been merged into both branches is to look at commit messages, so when doing it this way the commits squashed and with a different message. If thats too hard then at least list them all here so they are linked back on the main PR and I can click through.

**Comment by StrongMonkey** [comment]
@cbron Yeah next time I will seperate these. There are two different backport PR that touched the same code so doing it seperately will need to rebase and modified code, that's why I put it together this time.

**Review by cbron — APPROVED** [review]

**Pull request merged**